src/css/modules/_Steps.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 .coa-Steps__list {
-  & > ol {
+  & > ul {
```

Comment:
Oh snap, this might've been my bad. We should probably keep both `ul` and `ol`, but probably Joplin should be returning `ol` instead of `ul`

Reply:
truth. both cases would've made sense. I'd leave Joplin for now.